include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -1298,7 +1298,7 @@ FUNCTION(TF_NewStatus, TF_NewStatus, C_CC,
          ATTRS(NoUnwind))
 
 FUNCTION(TF_DeleteStatus, TF_DeleteStatus, C_CC,
-         RETURNS(),
+         RETURNS(VoidTy),
```

Comment:
good catch! I wonder if this is the reason behind the crash at https://bugs.swift.org/browse/SR-8983. If so, please feel free to resolve that bug. :-)

Reply:
oh cool, it does fix it.